roles/common/tasks/system-file-permissions.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Set permissions for system log directory recurisively
+  file: path=/var/log state=directory recurse=yes mode=0640
```

Comment:
this is getting removed because the directory already is up to standard. a false positive must have shown up